extensions/guacamole-auth-cas/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET" : ""
```

Comment:
Since in the current implementation the redirect to CAS is transparent, and the ticket is handled transparently, this just serves to make sure any passing UI components are essentially blank as the user gets redirected to the authentication page and back.  Maybe it's unnecessary...